packages/cdktf-cli/bin/cmds/handlers.ts

```diff
@@ -167,19 +173,15 @@ export async function diff(argv: any) {
   );
 }
 
-export async function get(argv: any) {
+export async function get(argv: { output: string; language: Language }) {
```

Comment:
Nice improvement, from type cast to type assertion

Reply:
yeah, I moved that upwards, because I started calling that method from elsewhere, so I wanted those assumptions to be in the arguments 😁